backend/src/test/java/capstone/storage/backend/ItemIntegrationTest.java

```diff
@@ -0,0 +1,481 @@
+package capstone.storage.backend;
+import capstone.storage.backend.exceptions.ExceptionMessage;
+import capstone.storage.backend.item.models.Item;
+import capstone.storage.backend.item.models.ItemResponse;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.MockWebServer;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.springframework.test.context.DynamicPropertySource;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import java.io.IOException;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+class ItemIntegrationTest {
+    @Autowired
+    private MockMvc mockMvc;
+    private static MockWebServer mockWebServer;
+    @Autowired
+    private ObjectMapper objectMapper;
```

Comment:
Bitte den Testcode so unabhängig wie möglich vom Production Code machen.

```suggestion
    private ObjectMapper objectMapper = new ObjectMapper();
```